server/controllers/request.js

```diff
@@ -14,22 +14,25 @@ const getRequests = async (req, res) => {
 // @route POST
 
 const createRequest = async (req, res) => {
-  req.body.userId = req.user.id;
-  const request = await Request.create(req.body);
+  const { sitterId, start, end } = req.body;
+  const userId = req.user.id;
+  const request = await Request.create({ userId, sitterId, start, end });
   res.send(request);
 };
 
 // @desc UPDATE : Update request with approved or decline
-// @route PUT
+// @route PATCH
 
 const updateRequest = async (req, res) => {
-  Request.findByIdAndUpdate({ _id: req.params.id }, req.body).then(
-    (request) => {
-      Request.findOne({ _id: req.params.id }).then((request) => {
-        res.send(request);
-      });
+  const { status } = req.body;

```

Comment:
Also validate this one

Reply:
@ezeilo-su , I think, this is from the previous commit. I already made the changes in this file.

Reply:
Same as my comment above about validtion